test/cli/commands_test.py

```diff
@@ -52,22 +52,23 @@ def invoke_assert_code(
     return result
 
 
-expected_output = """== [test/fixtures/linter/indentation_error_simple.sql] FAIL
```

Comment:
Many changes in this file. I generally updated these tests in one of two ways:
* `chdir` to the `test/fixtures/` directory, which now has a `.sqlfluff` file that specifies `dialect=ansi`
* Specify `--dialect=ansi` on the command line.